md-dao/src/main/java/com/wipro/ats/bdre/md/dao/LineageNodeDAO.java

```diff
@@ -87,6 +112,17 @@ public String insert(LineageNode lineageNode) {
         return id;
     }
 
+    public LineageNode getColNodeId(String colName, LineageNode tableNode) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00112)